### DISCUSSION TITLE
Mp4 video format

### DISCUSSION THREAD

**Discussion by texxnik** [jameszah/ESP32-CAM-Video-Telegram#4] [Ideas]
Hello, can I do that the command / clip would come to the telegram file mp4? What could be immediately without downloading to look directly in the telegram?

**Comment by jameszah** [comment]
It does not currently do an mp4.  I think I have figured out the format, but haven't added it yet.
I think when you send the mp4 to telegram is should display the first frame as picture, so you don't have to send the jpeg separately.
Stay tuned.
